pkg/reconciler/apiserversource/resources/oidc_rolebinding.go

```diff
@@ -19,16 +19,28 @@ package resources
 import (
 	"fmt"
 
+	"knative.dev/pkg/kmeta"
+
 	rbacv1 "k8s.io/api/rbac/v1"
 	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
 	v1 "knative.dev/eventing/pkg/apis/sources/v1"
 )
 
+// createOIDCTokenRoleName will return the name of the role for creating the JWT token
+func CreateOIDCTokenRoleName(source *v1.ApiServerSource) string {
```

Comment:
```suggestion
// GetOIDCTokenRoleName will return the name of the role for creating the JWT token
func GetOIDCTokenRoleName(source *v1.ApiServerSource) string {
```
nit: to not confuse with "create" here.